apps/mark/backend/src/app.test.ts

```diff
@@ -68,57 +70,6 @@ test('uses default machine config if not set', async () => {
   });
 });
 
-test('auth', async () => {
```

Comment:
I actually previously moved these tests to [app.auth.test.ts](https://github.com/votingworks/vxsuite/blob/main/apps/mark/backend/src/app.auth.test.ts) and forgot to remove the old ones still in this file